tests/conftest.py

```diff
@@ -89,3 +121,17 @@ def json_to_string(dictionary: dict[str, Any]) -> str:
         ensure_ascii=True,
         separators=(",", ":"),
     ).replace("\n", "")
+
+
+def log_spark_dataframe(df: DataFrame, *, truncate: bool = False, name: str = "") -> None:
+    """Logs the contents of a Spark DataFrame in tabular format.
+
+    Useful when Pytest is configured to capture only logs, so `df.show()` won't work.
```

Comment:
Isn't it better to configure `Pytest` to not only capture logs?

Reply:
Usually yes, but see explanation above